public/app/assetlibrary/item/item.html

```diff
@@ -4,17 +4,17 @@
 
   <!-- TITLE -->
   <div class="row assetlibrary-item-top">
-    <div class="col-xs-{{ me.is_admin ? 5 : (canManageAsset() ? 8 : 12) }} col-sm-{{ me.is_admin ? 7 : (canManageAsset() ? 10 : 12) }}">
+    <div class="col-xs-{{ asset.canDelete ? 5 : (canManageAsset() ? 8 : 12) }} col-sm-{{ asset.canDelete ? 7 : (canManageAsset() ? 10 : 12) }}">
```

Comment:
I think we've mostly been using the `can_delete` format in the REST API responses